conf/svcomp23.json

```diff
@@ -70,7 +70,8 @@
         "congruence",
         "octagon",
         "wideningThresholds",
-        "loopUnrollHeuristic"
+        "loopUnrollHeuristic",
+        "termination"
```

Comment:
This shouldn't change the svcomp23 conf, but the general svcomp one only.

Reply:
I changed the configs accordingly.